ios-app/UI/WebViewController.swift

```diff
@@ -188,10 +188,5 @@ class WebViewController: BaseWebViewController, WKWebViewDelegate {
 
     func onFinishLoadingWebView() {
         activityIndicator?.stopAnimating()
-        if (webView.canGoBack) {
-            navItem?.leftBarButtonItem = UIBarButtonItem(customView: backButton)
-        } else {
-            navItem?.leftBarButtonItem = nil
-        }
```

Comment:
We're removing this because if the web view doesn't have a previous page, the back button is removed from the navigation bar. However, users can still use it to navigate back to the previous page they came from.